app/js/controllers/main.js

```diff
@@ -34,6 +50,10 @@ export default class MainController extends BaseController {
         });
     }
 
+    this[p.speechController].start().then(() => {
+      console.log('Speech controller started');
+    });
```

Comment:
Maybe in the future we want to wait until the speech controller has started before starting the app to prevent nasty race conditions.
I don't think that's likely to happen though. What do you think?